swift/ql/src/queries/Security/CWE-094/UnsafeJsEvalGood.swift

```diff
@@ -0,0 +1,10 @@
+let webview: WKWebView
+let remoteData = try String(contentsOf: URL(string: "http://example.com/evil.json")!)
+
+...
+
+_ = try await webview.callAsyncJavaScript(
+    "alert(JSON.parse(data))",
```

Comment:
Is there a good reason to have `JSON.parse(data)` here rather than just `data`?

Other than that question I really like this example.  Its simple but plausible.

Reply:
Presumably the intent of the example is to read a remote JSON string, either by inlining into Js (bad) or parsing as JSON (good). Though perhaps alert() is not the best function to wrap this in.

Reply:
I'm happy with `alert`, its simple and the bad example is clearly vulnerable.  On reflection I think its the presence of `JSON.parse` is one example but not the other that bothers me just a little.  A reader could misunderstand that to be relevant to the safety of the good example compared to the bad one.